src/blocks/weather/met_no.rs

```diff
@@ -2,13 +2,15 @@ use super::*;
 
 type LegendsStore = HashMap<String, LegendsResult>;
 
-#[derive(Deserialize, Debug)]
-#[serde(tag = "name", rename_all = "lowercase")]
+#[derive(Deserialize, Debug, SmartDefault)]
+#[serde(tag = "name", rename_all = "lowercase", deny_unknown_fields, default)]
 pub struct Config {
     coordinates: Option<(String, String)>,
     altitude: Option<String>,
     #[serde(default)]
     lang: ApiLanguage,
+    #[default(12)]
+    forcast_hours: usize,
```

Comment:
Docs state that the default is `0`.

Reply:
The docs say 12 hours default for met.no and 0 hours default for owm (because of the extra api call). I can set the default to 12 though (and update the doc). I'll add in a check to see if the forecast needs to be fetched based on the format.

Reply:
> The docs say 12 hours default for met.no and 0 hours default for owm 

Sorry, didn't notice the difference.